lastpasslib/lastpasslib.py

```diff
@@ -653,7 +654,19 @@ def get_passwords(self):
             A list of password type secrets.
 
         """
-        return self.get_secrets(filter_='Password')
+        return self.get_secrets(filter_="Password")
```

Comment:
I think that the project uses single quotes consistently, let's not start diverging here for no benefit.